tests/test_quickbooks_online/test_views.py

```
@@ -147,3 +156,207 @@ def test_post_refresh_dimensions(api_client, test_connection):
+            data=nonexistent_payload_json,
+            content_type='application/json',
+            **{'HTTP_INTUIT_SIGNATURE': nonexistent_signature}
+        )
+
+    # Should still return 202 but no new records created
+    assert response.status_code == status.HTTP_202_ACCEPTED
+    assert QBOWebhookIncoming.objects.count() == initial_count
+
+    # Test 7: Exception handling should still return 202
+    with patch.dict(os.environ, {'QBO_WEBHOOK_TOKEN': webhook_token}):
+        with patch('apps.quickbooks_online.serializers.QBOWebhookIncomingSerializer.create') as mock_create:
```

Ruff (0.11.9)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)